xml/depl_nodes.xml

```diff
@@ -171,6 +171,11 @@
     barclamp that is not the admin node. The admin node is not added to the
     public network. So another node is needed that can be attached to the public
     network and appear in the designate default pool.
+
+    In High Availability deployment where Designate services are running in a cluster,
+    it is highly recommended that DNS services are running in the same cluster as well.
+    Therefore, DNS barclamp should be applied the the same nodes as the
```

Comment:
```suggestion
    Therefore, the DNS barclamp should be applied to the same nodes as the
```

Reply:
Why is it better to run DNS in the same cluster? I get that a DNS cluster is a good thing (:tm:) but what's the advantage of it being the same cluster?

Reply:
You're right. It doesn't have to be in the same cluster. But we usually group all the related services in the same cluster right? I can change it from "same cluster" to "a cluster". Please let me know.

Reply:
Ah ok, that makes sense. Yes, in CI we do, but who is to say what a customer might prefer? :wink: Let's just change it to "a cluster" as you suggest. Then we are definitely on the safe side :smile:

Reply:
done